pandas/tests/reshape/merge/test_merge_asof.py

```diff
@@ -1006,6 +1006,18 @@ def test_merge_datatype_error(self):
         with pytest.raises(MergeError, match=msg):
             merge_asof(left, right, on='a')
 
+    def test_merge_datatype_categorical_error(self):
+        """ Tests merge datatype mismatch error """
```

Comment:
Can remove docstring for test - we don't have these anywhere else

Reply:
Likewise. I do that for `test_merge_datatype_error` too?

Reply:
There are actually 4 other uses of docstrings in tests in this file. I convert them all to comments?

Reply:
I guess ignore for now but still remove this. Can have a follow up PR to make that module more consistent